aries-site/src/data/structure.js

```diff
@@ -307,10 +344,12 @@ export const structure = [
   {
     name: 'Accordion',
     description: 'Buttons are used to indicate actions that can be performed.',
-    previewImage: {
-      src: '/accordion-light.svg',
-      alt: 'HPE Accordion Preview',
-      fit: 'contain',
+    preview: {
+      image: {
```

Comment:
Looks like we are still using the image here rather than the preview component

Reply:
Since accordion was supplied to me in the svgs I was going to change this in the other PR since I have now both light and dark versions but either way we can change it here now as well

Reply:
Is there an issue representing this work? As long as its documented I am okay, but could get lost if someone else were to pick it up. There is a preview example file for it here: https://github.com/hpe-design/design-system/blob/7a8a3ab077002c817f0c6e6a479adf8d595821ee/aries-site/src/examples/cardPreviews/accordion.js
So it's a little hard to follow along from a review standpoint.

Reply:
I can document it! Im so glad you pointed out that file I can go ahead and delete that one

Reply:
Added to issue #589